app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -67,16 +66,18 @@ class ConversationViewModel @Inject constructor(
         }
     }
 
-    fun onMessageChanged(message: TextFieldValue) {
+    fun onMessageChanged(message: String) {
         conversationViewState = conversationViewState.copy(messageText = message)
     }
 
     fun sendMessage() {
+        val messageText = conversationViewState.messageText
+
+        conversationViewState = conversationViewState.copy(messageText = "")
         viewModelScope.launch {
-            val messageText = conversationViewState.messageText
             // TODO what if conversationId is null???
-            sendTextMessage(conversationId!!, messageText.text)
-            conversationViewState = conversationViewState.copy(messageText = messageText.copy(""))
+            sendTextMessage(conversationId!!, messageText)
```

Comment:
Shouldn't we make sure to invoke the use case only if the conversationId is not null?
```suggestion
            conversationId?.let { sendTextMessage(it, messageText) }
```